src/script/standard.h

```diff
@@ -73,6 +73,22 @@ class CNoDestination {
     friend bool operator<(const CNoDestination &a, const CNoDestination &b) { return true; }
 };
 
+struct PKHash : public uint160
+{
+    PKHash() : uint160() {}
+    explicit PKHash(const uint160& hash) : uint160(hash) {}
+    explicit PKHash(const CPubKey& pubkey);
```

Comment:
Inline definition might be more clear but I don't want to upset any strong c++ preferences out there.

Reply:
It's a mixed bag here(definition for other constructors are in .cpp), I'll leave as-is unless someone else chimes in